keras_cv/models/backbones/mobilenet_v3/mobilenet_v3_backbone.py

```diff
@@ -0,0 +1,518 @@
+            filters = filters * se_ratio.
+        activation: the activation layer to use.
+        expansion_index: integer, a unique identification if you want to use
+            expanded convolutions.
+        name: string, layer label.
+
+    Returns:
+        the updated input tensor.
+    """
+    if name is None:
+        name = f"inverted_res_block_{backend.get_uid('inverted_res_block')}"
+
+    shortcut = x
+    prefix = "expanded_conv/"
+    infilters = backend.int_shape(x)[channel_axis]
+
+    if expansion_index:
+        prefix = f"expanded_conv_{expansion_index}"
+
+        x = layers.Conv2D(
+            adjust_channels(infilters * expansion),
+            kernel_size=1,
+            padding="same",
+            use_bias=False,
+            name=prefix + "expand",
+        )(x)
+        x = layers.BatchNormalization(
+            axis=channel_axis,
+            epsilon=1e-3,
+            momentum=0.999,
+            name=prefix + "expand/BatchNorm",
+        )(x)
+        x = activation(x)
+
+    x = layers.DepthwiseConv2D(
+        kernel_size,
+        strides=stride,
+        padding="same" if stride == 1 else "valid",
+        use_bias=False,
+        name=prefix + "depthwise",
+    )(x)
+    x = layers.BatchNormalization(
+        axis=channel_axis,
+        epsilon=1e-3,
+        momentum=0.999,
+        name=prefix + "depthwise/BatchNorm",
+    )(x)
+    x = activation(x)
+
+    if se_ratio:
+        with custom_object_scope({"hard_sigmoid": apply_hard_sigmoid}):
+            x = cv_layers.SqueezeAndExcite2D(
+                filters=adjust_channels(infilters * expansion),
+                ratio=se_ratio,
+                squeeze_activation="relu",
+                excite_activation="hard_sigmoid",
+            )(x)
+
+    x = layers.Conv2D(
+        filters,
+        kernel_size=1,
+        padding="same",
+        use_bias=False,
+        name=prefix + "project",
+    )(x)
+    x = layers.BatchNormalization(
+        axis=channel_axis,
+        epsilon=1e-3,
+        momentum=0.999,
+        name=prefix + "project/BatchNorm",
+    )(x)
+
+    if stride == 1 and infilters == filters:
+        x = layers.Add(name=prefix + "Add")([shortcut, x])
+
+    return x
+
+
+v3Small = {
+    "stackwise_expansion": [1, 72.0 / 16, 88.0 / 24, 4, 6, 6, 3, 3, 6, 6, 6],
+    "stackwise_filters": [16, 24, 24, 40, 40, 40, 48, 48, 96, 96, 96],
+    "stackwise_stride": [2, 2, 1, 2, 1, 1, 1, 1, 2, 1, 1],
+}
+v3Large = {
+    "stackwise_expansion": [1, 4, 3, 3, 3, 3, 6, 2.5, 2.3, 2.3, 6, 6, 6, 6, 6],
+    "stackwise_filters": [
+        16,
+        24,
+        24,
+        40,
+        40,
+        40,
+        80,
+        80,
+        80,
+        80,
+        112,
+        112,
+        160,
+        160,
+        160,
+    ],
+    "stackwise_kernel_size": [1, 2, 1, 2, 1, 1, 2, 1, 1, 1, 1, 1, 2, 1, 1],
+}
+
+
+def stack_fn_v3small(x, kernel, activation, se_ratio, alpha=1.0):
```

Comment:
I think you can inline this into the model and just have parallel arrays of `kernel_size`, `activation` and `se_ratio`

Reply:
Addressed